normalizer/tests/test_https_endpoint.py

```
@@ -0,0 +1,98 @@
+        print(f"  順方向ジオコーディング: {endpoint_set['geocode']}")
+        try:
+            response = requests.get(
+                endpoint_set['geocode'],
+                params={"addr": test_address},
+                timeout=10
+            )
+            print(f"    ステータスコード: {response.status_code}")
+            if response.status_code == 200:
+                data = response.json()
+                if data and len(data) > 0:
+                    print(f"    [OK] レスポンス取得成功")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)